python/neuroglancer/lib.py

```diff
@@ -272,6 +300,11 @@ def min2(a, b):
 def clamp(val, low, high):
   return min(max(val, low), high)
 
+def eclamp(val, low, high):
```

Comment:
It's for when you would otherwise use clamp, but want it to error this time instead of just clamp the value it's out of range.